client/src/components/Sidebar/Sidebar.jsx

```diff
@@ -8,7 +8,7 @@ import { FeedbackButton } from '../FeedbackForm';
 import * as ROUTES from '../../constants/routes';
 import HistoryBadge from '../History/HistoryBadge';
 import './Sidebar.global.scss'; // XXX: Global stylesheet imported in component
-import styles from './Sidebar.module.scss';
+import styles from './Sidebar.module.css';
```

Comment:
Can this sidebar be updated to use the global component? I'm hesitant to approve this unless we can confirm that the new abstraction can actually replace the thing it's meant to.

Reply:
I agree. At least one existing sidebar should use the new sidebar.

<sub>P.S. [FP-1560](https://jira.tacc.utexas.edu/browse/FP-1560) exists to make all remaining sidebars use the component.</sub>